fili-core/src/main/java/com/yahoo/bard/webservice/metadata/LookupMetadataLoadTask.java

```diff
@@ -0,0 +1,124 @@
+// Copyright 2018 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.metadata;
+
+import com.yahoo.bard.webservice.application.LoadTask;
+import com.yahoo.bard.webservice.config.SystemConfig;
+import com.yahoo.bard.webservice.config.SystemConfigProvider;
+import com.yahoo.bard.webservice.data.dimension.DimensionDictionary;
+import com.yahoo.bard.webservice.data.dimension.impl.LookupDimension;
+import com.yahoo.bard.webservice.druid.client.DruidWebService;
+import com.yahoo.bard.webservice.druid.client.FailureCallback;
+import com.yahoo.bard.webservice.druid.client.HttpErrorCallback;
+import com.yahoo.bard.webservice.druid.client.SuccessCallback;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+/**
+ * Lookup Load task sends requests to Druid coordinator and returns list of configured lookup statuses in Druid.
+ */
+public class LookupMetadataLoadTask extends LoadTask<Boolean> {
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+
+    /**
+     * Location of lookup statuses on Druid coordinator.
+     */
+    public static final String LOOKUP_QUERY = "/lookups/status/__default";
```

Comment:
__default namespace is not always going to be where lookups are loaded.  We need to make a list of namespaces that is configurable, defaulting to __default if not configured.